tests/chainer_tests/links_tests/loss_tests/test_hierarchical_softmax.py

```diff
@@ -107,5 +107,21 @@ def test_to_cpu(self):
         self.assertTrue((f.paths == g.paths).all())
         self.assertTrue((f.codes == g.codes).all())
 
+    def _test_sampling(self, x):
+        x = chainer.Variable(x)
+        result = self.link.sampling(x)
+        self.assertEqual(len(result), x.shape[0])
+        for word_id in result:
+            self.assertEqual(type(word_id), int)
```

Comment:
And check if the result is in the valid range. I mean `self.assertLe(0, word_id)` and `self.assertLt(word_id, 5)`